src/app/components/feed/FeedDataPointsSection.js

```diff
@@ -34,26 +51,186 @@ const FeedDataPointsSection = ({
+              <FormattedMessage
+                defaultMessage="Anonymized unique user ID (example: 7299e0)"
+                description="Request field listed on save feed page, as part of the list of request fields available when the feed is sharing requests. 7299e0 doesn't need to be localized."
+                id="feedDataPoints.feedDataPointAnonymizedUserId"
+                tagName="li"
+              />
+              <FormattedMessage
+                defaultMessage="Date"
+                description="Request field listed on save feed page, as part of the list of request fields available when the feed is sharing requests."
+                id="feedDataPoints.feedDataPointDate"
+                tagName="li"
+              />
+              <FormattedMessage
+                defaultMessage="Platform (WhatsApp, Telegram, etc.)"
+                description="Request field listed on save feed page, as part of the list of request fields available when the feed is sharing requests."
+                id="feedDataPoints.feedDataPointPlatform"
+                tagName="li"
+              />
+              <FormattedMessage
+                defaultMessage="Language"
+                description="Request field listed on save feed page, as part of the list of request fields available when the feed is sharing requests."
+                id="feedDataPoints.feedDataPointRequestLanguage"
+                tagName="li"
+              />
+            </ul>
+            <div className="typography-body2">
+              <FormattedMessage
+                defaultMessage="Learn more about {mediaDataArticleLink}."
+                description="Footer of section on save feed page where a user can select that media and requests will be shared in this feed."
+                id="feedDataPoints.mediaContentFooter"
+                values={{
+                  mediaDataArticleLink: (
+                    <a href="https://help.checkmedia.org/en/articles/8542417-shared-feed-creation#h_7414a7ab59" rel="noopener noreferrer" target="_blank"> {/* FIXME: Put the correct link here */}
+                      <FormattedMessage
+                        defaultMessage="media & requests data"
+                        description="Text of the link to an article that explains media and requests data. This link is displayed on the media/requests data points section on the save feed page."
+                        id="feedDataPoints.mediaDataArticleLink"
+                      />
+                    </a>
+                  ),
+                }}
+              />
+            </div>
+          </div>
+          : null }
       </div>
     </div>
   );
 };
 
 FeedDataPointsSection.defaultProps = {
-  readOnly: false,
   enabled: false,
+  listId: null,
+  readOnly: false,
   onToggle: () => {},
 };
 
 FeedDataPointsSection.propTypes = {
-  content: PropTypes.node.isRequired,
   enabled: PropTypes.bool,
+  listId: PropTypes.number,
+  listType: PropTypes.string.isRequired,
```

Comment:
What are the possible values for listType?  
Perhaps something like `listType: PropTypes.oneOf(['media', 'article'])` would be a better propType definition